packages/components/addon/components/hds/pagination/nav/index.hbs

```diff
@@ -18,6 +18,22 @@
     </ul>
   {{/if}}
 
+  {{#if (eq this.type "truncated")}}
+    <ul class="hds-pagination-nav__page-list">
+      {{#each this.truncatedPages as |page|}}
+        {{#if (eq page "…")}}
+          <Hds::Pagination::Nav::Ellipses />
+        {{else}}
+          <Hds::Pagination::Nav::ButtonNumber
```

Comment:
general comment: I see the text and the buttons jump/move a bit when clicking next/prev buttons (because each character has a different width). A couple of options I can think of:
- apply a small min-width (removing the horizontal padding) - _probably would work only for 1 digits pages_
- apply a `font-variant-numeric: tabular-nums` style to the pages - _this slightly increases the kerning between numeric character, not sure if it's OK (maybe check with @cveigt)_

Reply:
I'll take a closer look, thanks

Reply:
I asked @cveigt  to weigh in on this. 

Personally, I think it would be worse to replace the in-between padding with min-width because then the space between numbers would then be inconsistent. I think adding `font-variant-numeric: tabular-nums` is an ok option to reduce jumping but overall the slight jumping doesn't bother me too much as to me it seems natural for the button sizes to change slightly to accommodate a different length of content.

(@cveigt you can see the different between "normal" and the "tabular-nums" variant on the page: https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric)

Reply:
The shift is an acceptable trade-off. There will be a small shift, but depending on what the current page is, this should happen only once in a while. When it happens, it'll likely be a few pixels ~6px and might not even be noticeable since the pages usually re-render when going from one page to another.

I don't think setting min-width or adding kerning would be the best solution. We likely won't find an ideal value that works for all numbers, and it'll be noticeable in most cases, unlike the shift.

Reply:
Thanks @cveigt I'm going to leave this as is then.